CHANGELOG.md

```diff
@@ -13,10 +13,11 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Removed
 
-- `django_subatomic.db.NotADecorator` is no longer a part of the public API.
-  It has been renamed to `_NotADecorator`.
-  This implementation detail was not intended for general use,
-  and may be removed in a future release.
+- `_contextmanager_without_decorator`, `_NonDecoratorContextManager`, `NotADecorator`.
+  These implementation details were not intended to be a part of our public API.
+  We have now implemented an alternative utility class, `contextmanager_only`.
+  This new utility class is not part of our public API.
+  IMO, it'd be nice if `contextlib` provided this functionality.
```

Comment:
I'd remove the editorialising here -- I don't think it adds anything for the users of this library.

Let me know when you've started the thread on discuss.python.org though 😉